package.json

```diff
@@ -21,6 +21,8 @@
   },
   "homepage": "https://github.com/canon20paul/Leaderboard#readme",
   "devDependencies": {
+    "@babel/core": "^7.19.3",
```

Comment:
- [ ] I wonder, what do you use for this file?

<img width="307" alt="Screen Shot 2022-10-13 at 07 30 32" src="https://user-images.githubusercontent.com/69011963/195485288-ef72eabe-3ccd-40dc-8f27-6853474107ca.png">